test/e2e/framework/cleanup.go

```diff
@@ -70,6 +73,7 @@ func RunCleanupActions() {
 	}()
 	// Run unlocked.
 	for _, fn := range list {
+		e2elog.Logf("Run Cleanup Action: %v", runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name())
```

Comment:
```suggestion
		e2elog.Logf("Running cleanup action: %v", runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name())
```

the change looks fine in general, but would it spam the logs a lot?

/approve

Reply:
We need to make a new choice
- no spamming logs
- know which action hang